### PR TITLE
House cleaning and few additions.

### PR DESCRIPTION
**Pull request by mulewile**
No users story. Added a few accessibility properties, fixed some bugs and added some sounds.

### CODE REVIEW

File: components/Form/index.js [hunk 1]
```diff
@@ -59,21 +63,28 @@ export default function Form({ onSubmit, isEditing, eventToEdit }) {
...
         />
         <label htmlFor="date">Date:</label>
         <input
           type="datetime-local"
           id="date"
           name="date"
-          defaultValue={eventToEdit?.date}
+          defaultValue={
+            eventToEdit?.date
+              ? new Date(eventToEdit.date).toISOString().slice(0, 16)
```
Comment (moonwave99) [inline components/Form/index.js:78]: I would create a `formatDateForInput` function and pass `eventToEdit.date ` to it - the less JS in JSX, the better!

File: pages/events/[id]/index.js [hunk 2]
```diff
@@ -63,6 +60,22 @@ export default function EventDetails() {
...
+    hour: "numeric",
+    minute: "numeric",
+    hour12: true,
+  };
+
+  const formattedDate = new Intl.DateTimeFormat(
+    navigator.language,
+    options
+  ).format(date);
+
```
Comment (moonwave99) [inline pages/events/[id]/index.js:78]: All this stuff belongs to a `formatDate` function as well - keep components as short as possible!

Bonus: you can reuse formatters in different components this way.

**Review by moonwave99 — COMMENTED** [review]
Just a couple of refactoring suggestions, otherwise LGTM

File: pages/events/[id]/index.js (hunk 2 above)
Comment (mulewile) [reply pages/events/[id]/index.js:78]: Thanks, will refactor accordingly.

**Pull request merged**